be/src/types/bitmap_value.h

```diff
@@ -93,7 +93,37 @@ class BitmapValue {
     // Construct a bitmap from given elements.
     explicit BitmapValue(const std::vector<uint64_t>& bits);
 
-    void add(uint64_t value);
+    // It is recommended to use batch writing to improve performance, such as add_many.
+    void add(uint64_t value) {
```

Comment:
Is it necessary to put this function's implementation in the header file?
To make compile speed faster, all the implementation should be in cpp source files unless this will impact the performance.

Reply:
I have tested it and it does affect performance.

Reply:
if put it in .cpp time is 5.066s, put it in .h time is 4.594s

Reply:
Both add() and _copy_on_write need to be placed in .h, so that _copy_one_write can be inlined

Reply:
if not set to .h

```
.L2148:
        .cfi_restore_state
        .loc 16 430 5 is_stmt 1
        .loc 16 431 9
        .loc 16 431 23 is_stmt 0
        call    _ZN9starrocks11BitmapValue14_copy_on_writeEv
```